build/multilocale.js

```diff
@@ -267,20 +374,21 @@ function L10nManager(gaiaDir,
 
     metas.defaultLanguage.setAttribute('content', self.defaultLocale);
 
-    var timestamp = getTimestamp(new Date());
-    metas.availableLanguages.setAttribute('content',
-      self.locales.map(function(loc) {
-        return loc + ':' + timestamp;
-      }).join(', '));
-
     var settingsFile = utils.getFile(self.gaiaDir, 'build', 'config',
         'common-settings.json');
     var settings = utils.getJSON(settingsFile);
 
     metas.appVersion.setAttribute('content', settings['moz.b2g.version']);
 
-    var str = utils.serializeDocument(doc);
-    utils.writeContent(file, str);
+    return getTimestamp().then(function(timestamp) {
+      metas.availableLanguages.setAttribute('content',
+        self.locales.map(function(loc) {
+          return loc + ':' + timestamp;
```

Comment:
Oops, I didn't notice this before.  I think we should have individual timestamps for each locale in `self.locales`, not a single identical one for all of them.  For pseudolocales, we should use the same timestamp as we do for en-US.